apps/src/code-studio/components/progress/teacherPanel/TeacherPanel.jsx

```diff
@@ -242,6 +244,12 @@ class TeacherPanel extends React.Component {
               )}
             </div>
           )}
+          {!!DCDO.get('family-name-features', false) && (
```

Comment:
Wow this is so convenient!

Reply:
Forreal! It was nice that Ryan already integrated it https://github.com/code-dot-org/code-dot-org/pull/53021/files so I could just reuse:)